.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '5 10 * * SUN'
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        perl-version:
+          - '5.16'
```

Comment:
Should we keep `5.14` around since that is the minimum perl version shown in [Makefile.PL](https://github.com/maxmind/webservice-paypal-paymentsadvanced/pull/29/files#diff-79b7ecad0595e3fce455f8d2e96f2b3f589d5f4b1b59ed0f92bb8972558f86b2R21)?

Reply:
I removed 5.14 as it was [failing](https://github.com/maxmind/webservice-paypal-paymentsadvanced/runs/3721480137?check_suite_focus=true). It appears the issue is the Mojolicious dependency. I think I'll just bump the Perl version as I don't know if there is much point in putting effort into getting it working on 5.14.